src/Bookings.js

```diff
@@ -1,18 +1,32 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
+import SearchResults from "./SearchResults.js";
 // import FakeBookings from "./data/fakeBookings.json";
 
+const [bookings, setBookings] = useState([]);
```

Comment:
Right now when I run your code I get an error saying:
```
Hooks can only be called inside of the body of a function component.
```
Line 6 needs to be inside the `Bookings` component.